src/lib.rs

```diff
@@ -187,7 +187,22 @@ When your Raft node is ticked and running, Raft should enter a `Ready` state. Yo
 `has_ready` to check whether Raft is ready. If yes, use the `ready` function to get a `Ready`
 state:
 
-```rust,ignore
+```
+# use raft::{
+#    Config,
+#    storage::MemStorage,
+#    raw_node::RawNode,
+# };
+
+# let config = Config {
+#    id: 1,
+#    peers: vec![1],
+#     ..Default::default()
+# };
+
+# let storage = MemStorage::default();
+# config.validate().unwrap();
```

Comment:
It's ok to skip the `validate` call. It doesn't have an effect unless you're using the returned value.